src/assignmentone/Item.java

```diff
@@ -0,0 +1,104 @@
+package assignmentone;
+
+
+/*
+ * This class is use to save details of item
+ * calulate final price
+ * display details of item
+ * getter fuctions of variable
+ */
+public class Item {
+    private final String name;
+    private final float price;
+    private final String type;
+    private final int quantity;
+    private  float tax;
+    private float finalPrice;
+
+    /*
+     * This is a constructor
+     *
+     * @param name     This is the first parameter to initialize name
+     * @param price    This is the second parameter to initialize price
+     * @param quantity this is the third parameter to initialize quantity
+     * @param type     This is the fourth parameter to initialize type
+     */
+    public Item(final String name, final float price, final int quantity, final String type) {
+        this.name = name;
+        this.price = price;
+        this.quantity = quantity;
+        this.type = type;
+    }
+
+    /*
+     * This is a method to sum up the tax and generate final price
+     */
+    public void totaling() {
+        tax = new Utils().calculateTax(type, price);
```

Comment:
why declaring the new instance of a util class ?

Reply:
ok